src/sidebar/components/test/annotation-test.js

```diff
@@ -598,29 +598,40 @@ describe('annotation', function() {
       });
     });
 
-    describe('#hasQuotes', function() {
-      it('returns false if the annotation has no quotes', function() {
+    describe('#quote', function() {
+      it('returns `null` if the annotation has no quotes', function() {
         var annotation = fixtures.defaultAnnotation();
         annotation.target = [{}];
         var controller = createDirective(annotation).controller;
 
-        assert.isFalse(controller.hasQuotes());
+        assert.isNull(controller.quote());
       });
 
-      it('returns true if the annotation has quotes', function() {
+      it('returns `null` if the annotation has selectors but no quote selector', function () {
+        var annotation = fixtures.defaultAnnotation();
+        annotation.target = [{
+          selector: [],
+        }];
+        var controller = createDirective(annotation).controller;
+
+        assert.isNull(controller.quote());
+      });
```

Comment:
I think if this `quote()` method were on a standalone annotation object that the annotation component used, rather than being part of the annotation component itself, then the tests would be simpler. For example I think `it('returns `null` if the annotation has no quotes'` could be a one-liner:

```javascript
assert.isNull(new Annotation({target: [{}]}).quote());
```

Multiply that across a large number of unit tests for a lot of different annotation-based logic and the test code will be a lot shorter and simpler, and also a lot faster as it's not creating AngularJS directives